docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -39,6 +39,79 @@ The Fairlearn package is most applicable to two kinds of harms:
+
+The concepts outlined in this glossary are relevant to sociotechnical contexts. 
+
+Construct Validity
+^^^^^^^^^^^^^^^^^^
+
+We explore the concept of construct validity, as described in in Measurement and Fairness [#4]_ , and as it relates to sociotechnical contexts. This in intended to illuminate the concept to readers who are working on solutions in sociotechnical contexts and may also be referenced in explorations that detail how to apply the concept to specific scenarios.
+
+Key Terms 
+~~~~~~~~~
+
+- **Sociotechnical context** – A context in which both societal and technical aspects shape who might be harmed by AI systems and how (see fairlearn.ai for more detailed definition).
+
+- **Unobservable theoretical construct** – An idea or concept that is unobservable and cannot be directly measured but must instead be inferred through observable measurements defined in a measurement model. 
+
+- **Measurement model** – The method and approach used to measure the unobservable theoretical construct.
+
+- **Construct reliability** – This is closely related to construct validity (detailed below) and can be thought of as the extent to which the measurements of an unobservable theoretical construct remain the same when measured at different points in time. A lack of construct reliability can either be due to a misalignment between the understanding of the unobservable theoretical construct and the methods being used to measure that construct, or to changes to the construct itself.
+
+- **Construct validity** – This can be thought of as the extent to which the measurement model can be used to produce meaningful and useful results.
+
+Key Term Examples  - Unobservable theoretical constructs and Measurement models
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- **Fairness** is an example of an unobservable theoretical construct. Several measurement models exist for measuring fairness, including demographic parity. See Fairlearn Metrics for more examples of measurement models for measuring fairness. 
+
+- **Teacher effectiveness** is an example of an unobservable theoretical construct. Common measurement models include student performance on standardized exams and qualitative feedback for the teacher’s students.
+
+- **Socioeconomic status** is an example of an unobservable theoretical construct. A common measurement model includes annual household income. 
+
+- **Patient benefit** is an example of an unobservable theoretical construct. A common measurement model involves patient care costs.
+
+**Note:** We cite several examples of unobservable theoretical constructs and measurement models for the purpose of explaining the key terms outlined above.  Please reference Measurement and Fairness for more detailed examples.
+ 
+What is Construct Validity? 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Though the Measurement and Fairness paper explores both Construct reliability and Construct validity, we focus our 
```

Comment:
I don't think its necessary to explain in detail why we defer from the paper. Instead, I'd suggest to re-write this paragraph to highlight in what way construct validity is particularly relevant to fairness related harms.

Reply:
Currently I have: We note that both play an important role in understanding fairness in sociotechnical 
contexts

Any suggestions on what a more detailed explanation of how construct validity is particularly relevant to fairness related harms? 

Having trouble doing this succinctly (since the hope was that the glossary itself would make the relationship clear)

Reply:
@hannawallach @mmadeo also curious to hear your thoughts on the above suggestion and how we might highlight in what what construct validity is particularly relevant to fairness related harms.

Reply:
Agreed @hildeweerts that we probably don't need to say why we differ from the paper, and also since there may be other PRs later addressing construct reliability.

Reply:
Feel free to use some of the language from the [issue](https://github.com/fairlearn/fairlearn/issues/769), e.g.,:

"Because fairness itself is unobservable (and is fundamentally contested), users of Fairlearn need to make decisions about what their theoretical understanding of fairness is, and how they operationalize that abstract concept via measurements of observable properties. At each stage, assumptions are made, which need to be tested.

Measurement modeling provides a language and a framework for making those assumptions explicit and testing them. Here, we explain some of the concepts that can be helpful in that process."